catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -60,6 +55,47 @@ protected void onCreate(Bundle savedInstanceState) {
 		}
 	}
 
+	private void applyAccessibilityStyles() {
+		SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
+		if (sharedPreferences.getBoolean("settings_accessibility_large_text", false)) {
+			getTheme().applyStyle(R.style.FontSizeLarge, true);
+		}
+		if (sharedPreferences.getBoolean("settings_accessibility_high_contrast", false)) {
+			getTheme().applyStyle(R.style.ContrastHigh, true);
+		}
+		if (sharedPreferences.getBoolean("settings_accessibility_element_spacing", false)) {
+			getTheme().applyStyle(R.style.SpacingLarge, true);
+		}
+		if (sharedPreferences.getBoolean("settings_accessibility_category_icons_high_contrast", false)) {
+			getTheme().applyStyle(R.style.CategoryIconContrastHigh, true);
+		}
+		if (sharedPreferences.getBoolean("settings_accessibility_category_icons", false)) {
+			getTheme().applyStyle(R.style.CategoryIconVisible, true);
+		}
+		if (sharedPreferences.getBoolean("settings_accessibility_category_icons_big", false)) {
+			getTheme().applyStyle(R.style.CategoryIconSizeLarge, true);
+		}
+
+		@AccessibilitySettingsFragment.FontStyle
+		String fontStyle = sharedPreferences.getString("settings_accessibility_font_style", "regular");
```

Comment:
Maybe `AccessibilitySettingsFragment.REGULAR` instead of `"regular"`?

Reply:
maybe... :-P